packages/in_app_purchase/lib/src/billing_client_wrappers/sku_details_wrapper.g.dart

```diff
@@ -20,15 +20,46 @@ SkuDetailsWrapper _$SkuDetailsWrapperFromJson(Map json) {
       sku: json['sku'] as String,
       subscriptionPeriod: json['subscriptionPeriod'] as String,
       title: json['title'] as String,
-      type: const SkuTypeConverter().fromJson(json['type'] as String),
+      type: _$enumDecode(_$SkuTypeEnumMap, json['type']),
```

Comment:
For some reason this generated with extra enum boilerplate and it shouldn't have.

Reply:
oh yeah I didn't know why this piece of code got changed, I'll discuss with you offline.